scheduler/src/common/interceptor/correlationId.interceptor.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  Injectable,
+  NestInterceptor,
+  ExecutionContext,
+  CallHandler,
+} from '@nestjs/common';
+import { Observable } from 'rxjs';
+import { tap } from 'rxjs/operators';
+import { Response } from 'express';
+import { ClsServiceManager } from 'nestjs-cls';
+
+@Injectable()
+export class CorrelationIdInterceptor implements NestInterceptor {
```

Comment:
This can be removed for the same reason as above. Only applicable for incoming http requests to scheduler.

Reply:
okay

Reply:
removed